ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -765,6 +765,9 @@ impl TraceBuilder {
         if inst.is_debug_call(self.aot_mod) {
             return Ok(());
         }
+        if AOT_MOD.func(*callee).name() == "yk_trace_basicblock"{
```

Comment:
A small comment why we need to do that would be helpful for our future selves, I believe.

Reply:
Added :point_right: 12e047f93d62f031a865b53816e29df0bc0dc06d